clients/docs/confluent-cli.rst

```diff
@@ -0,0 +1,256 @@
+
+.. code-block:: bash
+
+   alice   {"count":0}
+   alice   {"count":1}
+   alice   {"count":2}
+
+When you are done, press ``Ctrl-C``.
+
+#. View the :devx-examples:`consumer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/confluent-cli/confluent-cli-example.sh`.
+
+
+Avro And Confluent Cloud Schema Registry
+----------------------------------------
+
+.. include:: includes/client-example-schema-registry-3.rst
+
+.. Do we want to create a separate section, for steps 1 through 5, leave them as is
+   or put them under the "Produce Records" section?
+
+#. As described in the `Confluent Cloud
+   quickstart <https://docs.confluent.io/current/quickstart/cloud-quickstart/schema-registry.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   in the |ccloud| GUI, enable |ccloud| |sr|
+   and create an API key and secret to connect to it.
+
+#. Verify your |ccloud| |sr| credentials work from your host. In the output
+   below, substitute your values for ``<SR API KEY>``, ``<SR API SECRET>``, and
+   ``<SR ENDPOINT>``.
+
+   .. code-block:: text
+
+      # View the list of registered subjects
+      curl -u <SR API KEY>:<SR API SECRET> https://<SR ENDPOINT>/subjects
+
+      # Same as above, as a single bash command to parse the values out of $HOME/.confluent/java.config
+      curl -u $(grep "^schema.registry.basic.auth.user.info" $HOME/.confluent/java.config | cut -d'=' -f2) $(grep "^schema.registry.url" $HOME/.confluent/java.config | cut -d'=' -f2)/subjects
+
+#. View your local |ccloud| configuration file (``$HOME/.confluent/java.config``):
+
+   .. code-block:: bash
+
+      cat $HOME/.confluent/java.config
+
+#. In the configuration file, substitute values for ``<SR API KEY>``,
+   ``<SR API SECRET>``, and ``<SR ENDPOINT>``  as displayed in the following
+   example:
+
+   .. code-block:: bash
+
+      ...
+      basic.auth.credentials.source=USER_INFO
+      schema.registry.basic.auth.user.info=<SR API KEY>:<SR API SECRET>
+      schema.registry.url=https://<SR ENDPOINT>
+      ...
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      kafka-topics --bootstrap-server `grep "^\s*bootstrap.server" $HOME/.confluent/java.config | tail -1` --command-config $HOME/.confluent/java.config --topic test2 --create --replication-factor 3 --partitions 6
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+#. Run the `Confluent CLI
+   producer <https://docs.confluent.io/current/cli/command-reference/confluent-produce.html#cli-confluent-produce?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__,
+   writing messages to topic ``test2``, passing in arguments for:
+
+   -  ``--value-format avro``: use Avro data format for the value part of the
+      message
+   -  ``--property value.schema``: define the schema
+   -  ``--property schema.registry.url``: connect to the |ccloud| |sr| endpoint
+      http://
+
```

Comment:
@ybyzek  In the above list item it says "endpoint http://" Is a path missing here ?

Reply:
Updated